src/schemas/json/substreams-manifest-schema.json

```
@@ -0,0 +1,304 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "additionalProperties": false,
+  "description": "A Schema to support in making Substreams manifest files",
+  "name": "Substreams Manifest Schema",
+  "properties": {
+    "specVersion": {
+      "enum": ["v0.1.0"]
+    },
+    "package": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "pattern": "^([a-zA-Z][a-zA-Z0-9_]{0,63})$"
+        },
+        "version": {
+          "type": "string"
+        },
+        "url": {
+          "type": "string"
+        },
+        "doc": {
+          "type": "string"
+        }
+      },
+      "required": ["name", "version"],
+      "additionalProperties": false
+    },
+    "imports": {
+      "type": "object",
```

```suggestion
      "title": "imports",
      "description": "Imports\nhttps://substreams.streamingfast.io/reference-and-specs/manifests#imports"
      "type": "object",
```

Required fix (multiple violations): [add a title and an explanation](https://github.com/SchemaStore/schemastore/blob/master/CONTRIBUTING.md#best-practices).